workspaces/liferay-learn-workspace/client-extensions/liferay-learn-custom-element/src/components/Common/SectionCard.jsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Hi @Mylena-Rodrigues, 
Since the SectionCard.jsx and scss is being sent in the CoursesList PR, you don't need to send this files